src/components/TaskListView/TaskListView.js

```diff
@@ -0,0 +1,35 @@
+import './TaskListView.css'
+import { useState } from 'react'
+import TableRow from '../TableRow/TableRow'
+
+
+const TaskListView = (props) => {
+
+  const [tasks, setTasks] = useState(props.savedData)
+
+  const deleteTask = (id) => {
+    const allOtherTasks = tasks.filter(saved => saved.id !== id)
+    setTasks(allOtherTasks)
+  }
+
+  const rows = tasks.map((saved, index) => {
```

Comment:
Totally non blocking but I see that you use `saved` in many places to represent `savedTask`? 
For better readability, can we please call it that? `savedTask`

Reply:
Absolutely!! Great point!